.github/actions/setup/action.yml

```diff
@@ -13,11 +13,32 @@ runs:
             if [ ! -f "requirements.txt" ]; then
                 echo "python-gardenlinux-lib @ git+https://github.com/gardenlinux/python-gardenlinux-lib.git@${{ inputs.version }}" | tee -a requirements.txt
```

Comment:
The package name (before first `@`) should be `gardenlinux` these days.

Reply:
ah, right got it, thanks!